### PR TITLE
Feature/cal 027

### PR DESCRIPTION
**Pull request by alejandogodoy**

### CODE REVIEW

File: client/.eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...-RR-React\\client\\src\\App.js",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\Buildings.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\Companies.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\Technicians.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\layout\\header\\Header.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\layout\\sidenav\\Sidenav.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\boilers\\Boilers.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\boilers\\EditBoiler.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\boilers\\AddBoiler.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\boilers\\BoilersTable.jsx",[],"C:\\Users\\godoy\\Desktop\\CaldAR-RR-React\\client\\src\\components\\pages\\Boilers.jsx",[],{"ruleId":"53","replacedBy":"54"},{"ruleId":"55","replacedBy":"56"},"no-native-reassign",["57"],"no-negated-in-lhs",["58"],"no-global-assign","no-unsafe-negation"]
```
Comment (Ignacio-Sormani) [inline client/.eslintcache]: This file shouldn't be added

File: client/src/components/pages/boilers/Boilers.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  //DELETE BOILER
+  const deleteBoiler = (id) =>
+    setBoilers(boiler.filter((user) => user.id !== id));
+
+  return (
+    <div className={styles.container}>
+      <h1>Boilers</h1>
+      {edit ? (
+        <div>
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/Boilers.jsx]: Is this div necessary?

File: client/src/components/pages/boilers/EditBoiler.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            Edit Boiler
+          </button>
+          <button type="submit" onClick={() => props.setEditing(false)}>
+            Cancel
+          </button>
+        </form>
+      </Modal>
+    </>
+  );
+};
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/EditBoiler.jsx:73]: Add an empty line before the export default

File: client/src/components/pages/boilers/Boilers.module.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.container {
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/Boilers.module.css:1]: Css properties should appear in the correct order

File: client/src/components/pages/boilers/AddBoiler.jsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            onClick={handleSubmit}
+          >
+            Add new Boiler
+          </button>
+        </form>
+      </Modal>
+    </>
+  );
+};
+export default AddBoiler;
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/AddBoiler.jsx]: Add an empty line before the export default

**Review by Alfondalix — APPROVED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Pull request merged**